### PR TITLE
Task#2 - Authentication with passport-jwt and guards

### PR DESCRIPTION
**Pull request by warphole**
Good day! In this homework I have implemented authentication and guarding routes ny roles!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "eslint": "^8.10.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-node": "^4.1.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.28.0",
```
Comment (devbarden) [inline package.json:45]: react ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "eslint-plugin-react-hooks": "^4.3.0",
```
Comment (devbarden) [inline package.json:46]: react ? why is that here ?

File: src/loader/dbconnect.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const users = [];
+const accounts = [];
+const piggybanks = [];
+const transactions = [];
+
+function getUserByEmail(email) {
```
Comment (devbarden) [inline src/loader/dbconnect.js:6]: functions with operates with user should be stored at user controller

File: src/loader/dbconnect.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const users = [];
```
Comment (devbarden) [inline src/loader/dbconnect.js:1]: the data should be in json files or database

**Review by devbarden — COMMENTED** [review]
just a couple of issues

File: package.json (hunk 1 above)
Comment (asadbek2021) [reply package.json:45]: It has been installed as:
```npm i -D eslint prettier eslint-plugin-prettier eslint-config-prettier eslint-plugin-node eslint-config-node```
```npx install-peerdeps --dev eslint-config-airbnb```
```eslint --init```

I don't know why react plugins are there. I have done as the lector showed. How should I install plugins only for node?

File: package.json (hunk 2 above)
Comment (asadbek2021) [reply package.json:46]: the same answer as previous

File: src/loader/dbconnect.js (hunk 3 above)
Comment (asadbek2021) [reply src/loader/dbconnect.js:6]: ok

File: src/loader/dbconnect.js (hunk 4 above)
Comment (asadbek2021) [reply src/loader/dbconnect.js:1]: thanks
